Mathlib/FieldTheory/Galois.lean

```diff
@@ -420,8 +420,9 @@ theorem of_separable_splitting_field [sp : p.IsSplittingField F E] (hp : p.Separ
   intro K x hx hK
   simp only at *
   -- Porting note: need to specify two implicit arguments of `finrank_mul_finrank`
-  rw [of_separable_splitting_field_aux hp K (Multiset.mem_toFinset.mp hx), hK,
-    @finrank_mul_finrank _ _ _ _ _ _ _ K⟮x⟯.module _ K⟮x⟯.isScalarTower _]
+  letI := K⟮x⟯.module
```

Comment:
Can you add a comment saying that sooner or later those lines should be removed?

Reply:
For this one, there's already a porting note. I simply extracted the two arguments in the original `rw` to standalone `letI`s.